packages/back-end/src/models/FeatureModel.ts

```diff
@@ -640,6 +658,20 @@ export async function archiveFeature(
   return await updateFeature(context, feature, { archived: isArchived });
 }
 
+export async function syncEnvironmentSettings(
+  context: ReqContext | ApiReqContext,
+  feature: FeatureInterface,
+  sourceEnvironment: string,
+  destinationEnvironment: string
+) {
+  return await updateFeature(context, feature, {
+    environmentSettings: {
+      ...feature.environmentSettings,
+      [destinationEnvironment]: feature.environmentSettings[sourceEnvironment],
+    },
```

Comment:
This doesn't copy any rules in a current draft state but publishing that draft also doesn't clobber or otherwise affect the forked environment. That seems like reasonable and safe behavior to me but it's worth mentioning.

Reply:
One TODO for me before this PR is ready though - this leaves the settings in a weird state where we hit a sanity check error when fetching the feature later
```
`Published revision rules.${env} does not match feature ${org.id}.${feature.id}`
```
it still doesn't affect behavior as sdk connections and the feature page both show the correct values, but it's still worth fixing

Reply:
Man this was a convoluted behavior to untangle. I ended up actually making a new draft based on the current draft and then automatically publishing it so this won't happen anymore.